src/sssom_schema/schema/sssom_schema.yaml

```diff
@@ -405,14 +437,19 @@ slots:
       propagated: true
     examples:
       - value: http://purl.obolibrary.org/obo/mondo/releases/2021-01-30/mondo.owl
-        description: (A persistent Version IRI pointing to the Mondo version '2021-01-30')
+        description: A persistent Version IRI pointing to the Mondo version '2021-01-30'
   mapping_provider:
     description: URL pointing to the source that provided the mapping, for example
       an ontology that already contains the mappings, or a database from which it was derived.
     range: uri
     instantiates: sssom:Propagatable
     annotations:
       propagated: true
+    examples:
+      - value: https://www.ohdsi.org/
```

Comment:
I think the description needs to be updated here. These URIs don't actually point to resources, so what are you supposed to do with this? This is more of a comment string since I'm not sure there's a good way to actually capture this information unless it's supposed to point to a metadata file or the source data itself

Reply:
https://github.com/mapping-commons/sssom/issues/400